codespell_lib/data/dictionary.txt

```diff
@@ -20823,6 +21064,7 @@ thats->that's
 thats;->that's
 thay->they
 thck->thick
+thead->thread
```

Comment:
This trips up on:
https://www.w3schools.com/tags/tag_thead.asp

@larsoner any feelings of disable or remove?

Reply:
#1337